src/app/containers/FauxHeadline/index.jsx

```diff
@@ -11,6 +14,14 @@ import { GridItemConstrainedLarge } from '#lib/styledGrid';
 // missing CSS 'display: block;' in psammead branch `explicitly-set-h1-styles-display-and-font-weight`
 const FauxHeadline = props => <Headline as="strong" {...props} />;
 
+const StyledFauxHeadline = styled(FauxHeadline)`
+  padding: ${GEL_SPACING_QUAD} 0;
+
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    padding: 2.5rem 0;
```

Comment:
can we use our GEL spacing units to achieve this?

Reply:
The current GEL_SPACING options are (as far as I can tell):
```
var GEL_SPACING = "0.5rem";
var GEL_SPACING_DBL = "1rem"; // 16px
var GEL_SPACING_TRPL = "1.5rem";
var GEL_SPACING_HLF = "0.25rem";
var GEL_SPACING_QUAD = "2rem";
```
I saw 2.5rem hard coded in the `psammead-headings` comp so thought it was OK although didn't feel quite right...